rdkcentral/Lightning-UI-Components
# docs(Storybook): simplify Storybook sections to just Utilities and Components

## Description


The different sections in Storybook have proven to be more confusing than helpful, so condensing all stories into Docs, Utilities  (mixins) and Components. Leaving support for potential future "Patterns" and "Templates."

## References



## Testing


1. Confirm Storybook looks as expected with these new sections
2. Confirm the embedded canvases still work as expected (Docs tab of a component)
3. Confirm links to other components work in the docs

## Automation



## Checklist
